lib/tasks/install_secrets.rake

```diff
@@ -45,26 +45,40 @@ task :install_secrets, [] do
     end
   end
 
-  database_secrets = secrets.delete('database')
+  database_secrets = secrets.delete(:database)
+  write_yaml_file("config/database.yml", {
+    production: {
+      database: database_secrets[:name],
+      host: database_secrets[:url],
+      port: database_secrets[:port],
+      username: database_secrets[:username],
+      password: database_secrets[:password],
+      adapter: "postgresql",
+      encoding: "unicode",
+      pool: '<%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>'
```

Comment:
does 5 match what the needs of all puma's threads and workers? (curious if 5 has been reasoned out)

Reply:
good question.  I have no idea.  can I give one to you on the backburner?

Reply:
>If you are using the Puma web server we recommend setting the pool value to equal ENV['RAILS_MAX_THREADS']
https://devcenter.heroku.com/articles/concurrency-and-database-connections
so it should be 10.